src/components/hooks/useAdaptiveCard.js

```diff
@@ -27,7 +27,11 @@ export default function useAdaptiveCard(activityID) {
       cleanup = undefined;
     } else {
       const subscription = activitiesAdapter.getActivity(activityID)
-        .subscribe((activity) => setCard(activity.card));
+        .subscribe((activity) => {
+          const cardAttachment = activitiesAdapter.getAdaptiveCard(activity);
```

Comment:
this is not an attachment, an attachment has the structure `{contentType, content}`. This is just a card.